components/tensorflow-notebook-image/Dockerfile

```diff
@@ -20,15 +18,10 @@ ENV NB_USER jovyan
 ENV NB_UID 1000
 ENV HOME /home/$NB_USER
 ENV NB_PREFIX /
-# We prefer to have a global conda install
-# to minimize the amount of content in $HOME
-ENV CONDA_DIR=/opt/conda
-ENV PATH $CONDA_DIR/bin:$PATH
 
 # Export args as environment variables.
 # This is solely to make them available to install.sh
```

Comment:
Below you remove the call to `install.sh` so based on this comment do you still need these ENVs?

Reply:
Yes they are not needed except the last one. I have deleted them in a corresponding commit.